include/bluetooth/conn.h

```diff
@@ -334,6 +334,12 @@ typedef enum __packed {
  */
 int bt_conn_set_security(struct bt_conn *conn, bt_security_t sec);
 
+/** @brief Get security level for a connection.
+ *
+ *  @return Connection security level
+ */
+bt_security_t bt_conn_get_security(struct bt_conn *conn);
```

Comment:
Maybe move below the __deprecated declaration of the old bt_conn_security?

Reply:
For what reason?